src/main/java/com/adyen/Client.java

```diff
@@ -59,7 +59,10 @@
     public static final String ENDPOINT_PROTOCOL = "https://";
     public static final String POS_TERMINAL_MANAGEMENT_ENDPOINT_TEST = "https://postfmapi-test.adyen.com/postfmapi/terminal";
     public static final String POS_TERMINAL_MANAGEMENT_ENDPOINT_LIVE = "https://postfmapi-live.adyen.com/postfmapi/terminal";
+    public static final String DATA_PROTECTION_ENDPOINT_TEST = "https://ca-test.adyen.com/ca/services/DataProtectionService";
+    public static final String DATA_PROTECTION_ENDPOINT_LIVE = "https://ca-live.adyen.com/ca/services/DataProtectionService";
     public static final String POS_TERMINAL_MANAGEMENT_VERSION = "v1";
+    public static final String DP_VERSION = "v1";
```

Comment:
```suggestion
    public static final String DATA_PROTECTION_VERSION = "v1";
```
minor, just to align with your latest change